src/vumi2/applications/static_reply/static_reply.py

```diff
@@ -0,0 +1,39 @@
+from attrs import define
+
+from vumi2.messages import Event, Message, Session
+from vumi2.workers import BaseConfig, BaseWorker
+
+
+@define(kw_only=True)
+class StaticReplyConfig(BaseConfig):
+    transport_name: str
+    reply_text: str
+
+
+class StaticReplyApplication(BaseWorker):
+    """
+    A static reply application
+
+    It will reply to any inbound message with the configured reply text
```

Comment:
Use the present tense instead of the future tense, and preferably full sentences: "Replies to any inbound message with the configured reply text."